crates/op-rbuilder/README.md

```diff
@@ -77,34 +75,32 @@ git checkout op-rbuilder
 - Windows: `{FOLDERID_RoamingAppData}/reth/`
 - macOS: `$HOME/Library/Application Support/reth/`
 
-3. Run a clean OP stack in the `optimism` repo:
-
-```bash
-make devnet-clean && make devnet-down && make devnet-up
-```
-
-4. Run `op-rbuilder` in the `rbuilder` repo on port 8547:
+3. Run `op-rbuilder` in the `rbuilder` repo on port 4444:
 
 ```bash
-cargo run -p op-rbuilder --bin op-rbuilder --features optimism -- node \
-    --chain ../optimism/.devnet/genesis-l2.json \
-    --http \
-    --http.port 8547 \
-    --authrpc.jwtsecret ../optimism/ops-bedrock/test-jwt-secret.txt \
+cargo run -p op-rbuilder --bin op-rbuilder -- node \
+    --chain $HOME/.playground/devnet/l2-genesis.json \
+    --http --http.port 2222 \
+    --authrpc.port 4444 --authrpc.jwtsecret $HOME/.playground/devnet/jwtsecret \
+    --port 30333 --disable-discovery \
     --metrics 127.0.0.1:9001 \
-    --rollup.builder-secret-key ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+    --rollup.builder-secret-key ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \
+    --trusted-peers enode://3479db4d9217fb5d7a8ed4d61ac36e120b05d36c2eefb795dc42ff2e971f251a2315f5649ea1833271e020b9adc98d5db9973c7ed92d6b2f1f2223088c3d852f@127.0.0.1:30304
 ```
 
-5. Init `mev-flood`:
+4. Init `contender`:
 
 ```bash
-docker run mevflood init -r http://host.docker.internal:8547 -s local.json
+git clone https://github.com/flashbots/contender
+cd contender
+contender setup 
+cargo run -- setup ./scenarios/simple.toml http://localhost:2222
```

Comment:
When I run that I get:

```
opening DB at /home/denis/.contender/contender.db
no DB found, creating new DB
generating seed file at /home/denis/.contender/seed
No private keys provided. Using default private keys.
Error: "User account 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 has insufficient balance to fund all accounts. Have 0.000000000000000000, needed 0.010046200000000000. Chain ID: 13"
```

Could you please fix to use a proper account or deposit?

Reply:
@therustmonk are you running this [PR](https://github.com/flashbots/builder-playground/pull/103) of the builder-playground? The L2 accounts currently aren't funded by default until thats merged

Reply:
Thank you! It works with that PR.